disabled/blockchainHookCounter.go

```diff
@@ -0,0 +1,30 @@
+package disabled
+
+import vmcommon "github.com/ElrondNetwork/elrond-vm-common"
+
+// BlockChainHookCounter is a disabled implementation of blockChainHookCounter interface
```

Comment:
this is probably BlockChainHookCounter interface from vmcommon

Reply:
done